### ISSUE TITLE
How to change the gap between the images?

### ISSUE THREAD

**Issue by cnichte** [bep/gallerydeluxe#33]
How to change the gap between the images?

**Comment by bep** [comment]
I don't think you currently can do that (or at least not easily).

I notice that Pig (the JS library in use) has a ´spaceBetweenImages` which I have hardcoded to 1, but we could export that option to the Hugo config, I guess.

**Comment by cnichte** [comment]
Yep, I've found that too :-D  there are a bunch of options. Would be cool if they all went into the Hugo config and were passed to `pig.js`:

```js
var options = {
  containerId: 'pig',
  classPrefix: 'pig',
  figureTagName: 'figure',
  spaceBetweenImages: 8,
  transitionSpeed: 500,
  primaryImageBufferHeight: 1000,
  secondaryImageBufferHeight: 300,
  thumbnailSize: 20
  }
```

**Comment by bep** [comment]
I wouldn't mind just adding `pigOptions` to the config struct and just pass that on to Pig. Pull request would be welcomed.

**Comment by cnichte** [comment]
Cloned the repo and added the Change allready :-D

**Comment by cnichte** [comment]
Hi, I have no idea if I have done everything right. I can't create a pull request with my changes.

If you want to see how I used the galleries deluxe: [binary-voids.de/](https://binary-voids.de/)

- improved the navigation
- added theme support, and a toggle to switch to dark mode

Have a good start into the new year!

**Comment by bep** [comment]
>Have a good start into the new year!

Same to you. Some really great looking galleries at binary-voids.de. 

I will look into getting your suggestions into the theme ... next year. I have been busy doing nothing this xmas.
